src/sales/shipments-create.md

```diff
@@ -28,14 +28,45 @@ For each line item in the order, modify the **Qty to Ship** as needed.
 
 ## Shipping Information
 
-1. Click <span class="btn">Add Tracking Number</span>.
+### Method 1: Using the order page
+
+1. On the _Admin_ sidebar, go to **Sales** > **Orders**.
+
+1. In the **Action** column for the selected order, click **View**.
+
+1. Click **Ship**.
+
+1. Scroll down to the _Payment & Shipping Method_ block and click **Add Tracking Number**.
+
+1. Set **Carrier**:
+
+    - Custom Value;
```

Comment:
I think we can remove the punctuation at the end of these list items.

Reply:
thanks, done ✅